modules/oe_webtools_analytics/tests/src/Functional/ConfigurationTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_webtools_analytics\Functional;
+
+use Drupal\oe_webtools_analytics\AnalyticsEventInterface;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests that the configured settings are correctly output in the page.
+ *
+ * @group oe_webtools_analytics
+ */
+class ConfigurationTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'config',
+    'system',
+    'oe_webtools',
+    'oe_webtools_analytics',
+  ];
+
+  /**
+   * Tests if the configuration for the Webtools library is present on the page.
+   *
+   * @throws \Behat\Mink\Exception\ExpectationException
```

Comment:
This is not needed.

Reply:
I have a question regarding your comment @brummbar.

I guess @pfrenssen added this because the method throws an exception at a certain point and probably PHPStorm suggested to add it.
Is it a so bad thing to add it in the phpdoc? Why would we remove this?

Reply:
Hmm yeah this is actually quite pointless in tests to say which exception causes the tests to fail. This is left over from an earlier iteration where code that throws this exception was in a separate protected method. It looks like this code doesn't  even made it to the final version. I'll remove it.